RFCs/FS-1090-Generic-struct-type-whose-fields-are-all-unmanaged-types-is-unmanaged.md

```diff
@@ -0,0 +1,198 @@
+# F# RFC FS-1088 - (Generic struct type whose fields are all unmanaged types is unmanaged)
+
+The design suggestion [Generic struct type whose fields are all unmanaged types is unmanaged](https://github.com/fsharp/fslang-suggestions/issues/692) has been marked "approved in principle".
+This RFC covers the detailed proposal for this suggestion.
+
+* [x] Approved in principle
+* [x] [Suggestion](https://github.com/fsharp/fslang-suggestions/issues/692)
+* [ ] Implementation: [In progress](https://github.com/dotnet/fsharp/pull/6064)
+
+# Summary
+
+[summary]: #summary
+
+Allow generic structs to be `unmanaged` at construction if all fields are `unmanaged` types.
+
+# Motivation
+
+[motivation]: #motivation
+
+* Make it easier to author low level interop code.
```

Comment:
Although this will primarily be for calling APIs that likely do some native interop, it could be possible to be 100% F# code. So I think this is probably better as "low-level code", since that encapsulates both.